src-docs/src/views/tool_tip/tool_tip.js

```diff
@@ -56,6 +56,19 @@ export default () => (
 
     <EuiSpacer />
 
+    <p>
+      <EuiToolTip
+        position="top"
+        content="Here is some tooltip text"
+        display="block">
+        <EuiLink href="#">
+          I am a block level tooltip, applied to the span anchor
+        </EuiLink>
```

Comment:
This example a little weird to me because the text isn't very long so the tooltip can show up when hovering over the white space to the right of the text. A common use-case here for needing `display: block` is when it's wrapped around a `fullWidth` button or input. So maybe change this to:

```suggestion
        <EuiButton fullWidth href="#">
          I am a block level tooltip, applied to a fullWidth button
        </EuiLink>
```

Reply:
Your example was missing the closing tag, so I made this one separately.